web/src/components/dashboard/SidebarCardsContainer.test.tsx

```diff
@@ -91,6 +91,23 @@ describe("<SidebarCardsContainer />", () => {
       expect(screen.getByText("WelcomeCardContent")).toBeInTheDocument();
     });
 
+    it("renders the welcome-up-and-running card in the top card section", () => {
+      const sidebarCards = {
+        "welcome-up-and-running": generateSidebarCardContent({
+          id: "welcome-up-and-running",
+          contentMd: "WelcomeUpAndRunningCardContent",
+        }),
+      };
+
+      useMockUserData({
+        preferences: generatePreferences({ visibleSidebarCards: ["welcome-up-and-running"] }),
+      });
+
+      renderPage({ sidebarCards });
+      const topCardSection = screen.getByTestId("top-cards");
+      expect(within(topCardSection).getByTestId("welcome-up-and-running")).toBeInTheDocument();
+    });
```

Comment:
I don't think I would add this test. If I'm understanding it correctly, the card weight determines the ordering, right? And the weight is controlled by content, and could be changed by the content team at any point. We don't want to write a test that would fail if they change that "weight: 1" line to "weight: 10", which I believe this would do?